local_planner/test/test_local_planner_node.cpp

```diff
@@ -0,0 +1,55 @@
+#include <gtest/gtest.h>
+
+#include "../include/local_planner/local_planner_node.h"
+
+using namespace avoidance;
+
+TEST(LocalPlannerNodeTests, failsafe) {
```

Comment:
This looks great, hopefully in the future we can add more tests here to make sure we are subscribed to and handling inputs correctly.